process/smartContract/process.go

```diff
@@ -1127,21 +1034,27 @@ func (sc *scProcessor) ProcessSmartContractResult(scr *smartContractResult.Smart
 }
 
 func (sc *scProcessor) processSimpleSCR(
-	scr *smartContractResult.SmartContractResult,
-	dstAcc state.UserAccountHandler,
+	scResult *smartContractResult.SmartContractResult,
+	destinationScAccount state.UserAccountHandler,
```

Comment:
I think the destination account doesn't have to be a SC in this function (it can be a user account as well). The parameter should be named `destinationAccount` instead.

Reply:
it is too long for the name. dstAcc is enough.

Reply:
Renamed to `dstAcc`.